core/trino-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -327,6 +347,7 @@ class StageSummary extends React.Component {
 
         const scheduledTimes = stage.tasks.map(task => parseDuration(task.stats.totalScheduledTime));
         const cpuTimes = stage.tasks.map(task => parseDuration(task.stats.totalCpuTime));
+        const rawInputDataSizes = stage.tasks.map(task => parseDataSize(task.stats.rawInputDataSize));
```

Comment:
We should show physicalInputSize for source stage and internalNetworkInputDataSize for non-source stages. I think rawInputDataSize is something we want to eventually remove.
cc: @sopel39